projects/view/client/singleProject/singleProjectDetailed.css

```diff
@@ -0,0 +1,645 @@
+/*  SECTIONS  */
```

Comment:
This file contains redundant column classes. Please use [Bootstrap grid classes](http://getbootstrap.com/css/#grid) and remove these definitions.